torch/ao/quantization/_quantize_pt2e.py

```diff
@@ -79,6 +80,13 @@ def prepare_pt2e_quantizer(
     _rearrange_weight_observer_for_decomposed_linear(model)
     return model
 
+# TODO: update this to prepare_qat_pt2e
+def prepare_qat_pt2e_quantizer(
+    model: GraphModule,
+    quantizer: Quantizer,
+):
+    return prepare_pt2e_quantizer(model, quantizer, is_qat=True)
```

Comment:
I feel we should probably not call the PTQ version of this api, but instead just do the same thing as `prepare_pt2e_quantizer` here but change `_fuse_conv_bn_` to `_fuse_conv_bn_qat`

Reply:
Sounds good. However, we will need to do `_fuse_conv_bn_qat` between annotate and prepare. Should I move annotate out of prepare?